### ISSUE TITLE
Full batch mode skips over jobs loaded from a list

### ISSUE THREAD

**Issue by hlc94** [Nandaka/DanbooruDownloader#145]
So recently I've been using the save/load batch list feature for keeping track on some artists I follow on Danbooru. I made a list first using the Jan 31st version of the app (this was after September 22nd but I had no idea it was released yet), but haven't loaded and used it in the same version after completing the jobs and saved the batch.
Then I found out about Sept 22nd update and downloaded it. I loaded the batch list I made before and apparently it just skips over jobs without even checking the directory for existing/skippable files. I thought maybe batch lists from different versions could be the cause so I remade the entire jobs list manually and saved it with the Sept 22nd version. When I started the download right as I was manually adding them, the app properly checks the directories for existing files. But when I loaded the batch list afterwards, the same thing apparently happened.

tl;dr manually added jobs are properly checked for existing files, while saved and loaded ones are skipped, even though there clearly were new uploads on the artists' Danbooru page.

I appreciate any help you can give.

List made using Jan 31st version (granted there was one artist here that apparently changed names on Danbooru)
![image](https://user-images.githubusercontent.com/23534909/32299991-2ccb5d0e-bf8a-11e7-9b87-3d18c5fd4b5b.png)

List made using Sept 22nd version
![image](https://user-images.githubusercontent.com/23534909/32300058-703c1f88-bf8a-11e7-83be-536288583738.png)

Here are the log files when it happened
[Batch Download on 2017-10-30.txt](https://github.com/Nandaka/DanbooruDownloader/files/1435838/Batch.Download.on.2017-10-30.txt)
[Batch Download on 2017-11-01.txt](https://github.com/Nandaka/DanbooruDownloader/files/1435839/Batch.Download.on.2017-11-01.txt)
[log-2017-10-30.txt](https://github.com/Nandaka/DanbooruDownloader/files/1435840/log-2017-10-30.txt)
[log-2017-11-02.txt](https://github.com/Nandaka/DanbooruDownloader/files/1435841/log-2017-11-02.txt)
[log-2017-10-31.txt](https://github.com/Nandaka/DanbooruDownloader/files/1435842/log-2017-10-31.txt)
[log-2017-11-01.txt](https://github.com/Nandaka/DanbooruDownloader/files/1435843/log-2017-11-01.txt)

**Comment by Nandaka** [comment]
the tagging is depends on the tags.xml data. if the artist is not categorized properly, then it will be treated as unknown/general tag.

for the batch job, can you upload the xml file?

**Comment by hlc94** [comment]
[artist_list.txt](https://github.com/Nandaka/DanbooruDownloader/files/1441546/artist_list.txt)

Converted to txt because github doesn't support xml in the comments.

Thanks again for looking into this.

**Comment by Nandaka** [comment]
I think it is because of this entry, can you change it to false manually and re test?

![image](https://user-images.githubusercontent.com/836867/33412818-cd45d288-d5c8-11e7-9e76-dc45dd56484e.png)

**Comment by hlc94** [comment]
Nice, it works. Thanks! 

One question though, how come it tagged like over the first half of my list as completed (true) and leaves the rest incomplete (false) even though they've all finished downloading at least once in previous sessions?

**Comment by Nandaka** [comment]
That I'm not sure. By right if you save it when everything was done, it should be marked as completed.

Anyway, I think the isComplete flag shouldn't be stored in the list.

**Comment by hlc94** [comment]
I personally prefer it that way, or any other means that can help me recheck the list on a regular basis to get updates. Up to you though.

Thanks again for the help, I'm doing my list recheck atm haha.
